src/app/units/states/tasks/viewer/directives/f-task-sheet-view/f-task-sheet-view.component.html

```diff
@@ -1,12 +1,12 @@
 <div class="w-full h-full overflow-auto">
-  @switch (taskDef) { @case (taskDef) { @if (taskDef) {
+  @switch (taskDef) { @case (taskDef) { @if (taskDef && taskDef.hasTaskSheet) {
```

Comment:
I think this could be simplified to:

`@if (taskDef?.hasTaskSheet)`

Reply:
Yep sounds good, will make change

Reply:
Change made